ChebsNecromancy/Structures/Phylactery.cs

```diff
@@ -51,9 +63,60 @@ public static void CreateConfigs(BasePlugin plugin)
                     new ConfigurationManagerAttributes { IsAdminOnly = true }));
         }
 
-        public bool ConsumeFuel()
+        public static void ConfigureRPC()
+        {
+            PhylacteryCheckRPC = NetworkManager.Instance.AddRPC("PhylacteryCheckRPC",
```

Comment:
Configure the server version only on the server, and the client version only on the client.